generic_config_updater/generic_updater.py

```diff
@@ -122,6 +125,14 @@ def replace(self, target_config):
 
         self.logger.log_notice("Config replacement completed.")
 
+    @staticmethod
+    def switchport_mode_remove(config):
+        if 'PORT' in config:
+            for port, port_data in config['PORT'].items():
+                if 'mode' in port_data:
+                    del config['PORT'][port]['mode']
```

Comment:
Please extract constants for string 'mode' and 'PORT'

Reply:
updated as suggested,thanks